lib/graphql/schema.ts

```diff
@@ -3,7 +3,7 @@ export default `
     meta: Meta!
     applicants(filter: ApplicantsFilter): QueryApplicantsResult
     applicant(id: ID!): Applicant
-    employees: [Employee!]
+    employees(filter: EmployeesFilter): [Employee!]
```

Comment:
I tried creating a `QueryEmployeesResult` GraphQL type and made this line `employees(filter: EmployeesFilter): QueryEmployeesResult` but ran into some issues so I abandoned that.

 `QueryEmployeesResult` GraphQL type I initially created
```
type QueryEmployeesResult {
    result: [Employee!]!
  }
  ```
  
Is this approach ok?

Reply:
This approach is perfectly fine!